pkg/templates/imagepullpolicy/internal/params/params.go

```diff
@@ -0,0 +1,7 @@
+package params
+
+// Params represents the params accepted by this template.
+type Params struct {
+	// list of bad image pull policy
```

Comment:
Add comment lines saying `// +noregex` and `// +notnegatable` to signal that we're not allowing regexes and negations here.